.gitignore

```diff
@@ -1 +1,7 @@
-node_modules/
+node_modules
+.DS_Store
+
+# these cause more harm than good
+# when working with contributors
+package-lock.json
```

Comment:
What do we think about also removing `package-lock.json` and `yarn.lock` from the gitignore? So that one notices if a lock file is erroneously being used?

Reply:
@LinusU I have no strong opinion either way. I do like your thinking though. I do think we should align this one with https://github.com/standard/eslint-config-standard/blob/master/.gitignore